### PR TITLE
Enhanced tiobench.py, made it to run without any external disk pass

### PR DESCRIPTION
**Pull request by Naresh-ibm**
*Script was disk depend
* Now it is independent of disk and run on give directory or source
 directory
* Added tearDown function too

Signed-off-by: Naresh Bannoth <nbannoth@in.ibm.com>

### CODE REVIEW

**Comment by Naresh-ibm** [comment]
[tiobench_without_disk.txt](https://github.com/avocado-framework-tests/avocado-misc-tests/files/844390/tiobench_without_disk.txt)

[tiobench_with_disk.txt](https://github.com/avocado-framework-tests/avocado-misc-tests/files/844396/tiobench_with_disk.txt)

File: io/disk/tiobench.py [hunk 1]
```diff
@@ -64,30 +64,42 @@ def test(self):
...
                                                         threads, size,
                                                         num_runs))
 
+    def tearDown(self):
+
+        '''
+        Cleanup of disk used to perform this test
+        '''
+        if self.disk is not None:
+            self.log.info("Unmounting directory %s", self.target)
```
Comment (PraveenPenguin) [inline io/disk/tiobench.py]: i think message can be like unmounting disk 

`self.log.info("Unmounting Disk  %s", self.disk)`

Comment (PraveenPenguin) [inline io/disk/tiobench.py]: if you are going to follow this than please make self.target as local variable

File: io/disk/tiobench.py [hunk 2]
```diff
@@ -64,30 +64,42 @@ def test(self):
...
-        part_obj.unmount()
-        self.log.info("creating file system")
-        part_obj.mkfs(fstype)
-        self.log.info("Mounting disk %s on directory %s", disk, target)
-        part_obj.mount()
+        if self.disk is not None:
+            self.part_obj = Partition(self.disk, mountpoint=self.target)
+            self.log.info("Unmounting disk/dir before creating file system")
+            self.part_obj.unmount()
+            self.log.info("creating file system")
```
Comment (nasastry) [inline io/disk/tiobench.py]: Can you please add the kind of file system you are creating?
Example: `"Creating file system %s", fstype`

File: io/disk/tiobench.py (hunk 1 above)
Comment (nasastry) [inline io/disk/tiobench.py]: In this PR also can try 
`"Unmounting disk %s on directory %s", self.disk, self.scratch_dir`

**Comment by Naresh-ibm** [comment]
amended the changes suggested. please review it and let me know any further changes requires.

**Pull request merged**